### DISCUSSION TITLE
Is it possible to change the numbers format in the data importer

### DISCUSSION THREAD

**Discussion by snailcatcher** [firefly-iii/firefly-iii#8413] [Firefly III] [answered]
Hi there,

My bank uses dots as separator for 1000.
The data importer converts my amount `1.100` in `1,10` so way less than it should be.
Is there a way to fix this (e.g. a number format, like the date format)?

**Comment by JC5** [comment, marked as answer]
Not necessary! You found a bug that is fixed in the next release of the data importer.
